### DISCUSSION TITLE
now that --experimental-network-imports has been removed, is there a work around to import files from a server?

### DISCUSSION THREAD

**Discussion by patrick99e99** [nodejs/help#4618] [Q&A] [answered]
I have some server code for a project which is making use of external imports:
```
import { SomeClass } from 'http://localhost:8000/some-class.js';
```

I upgraded node today and saw the `--experimental-network-imports` option has been removed, so I am wondering is there any alternate way to accomplish what I am doing, or is it just completely impossible now and I need to stick to using node 21 or earlier?

**Comment by avivkeller** [comment]
You could fetch the data, convert it to a Data URL, and dynamically import the data URL?

**Comment by aduh95** [comment, marked as answer]
You can use a loader hook, see https://nodejs.org/api/module.html#import-from-https

**Comment by huuthang201** [comment]
thanks
